.github/workflows/ci.yml

```diff
@@ -149,18 +149,15 @@ jobs:
           RUST_BACKTRACE: 1
         run: nix develop -L --no-update-lock-file --command just impure-tests
 
+      # This differs from `nix-build-bats-tests` in that:
+      #   - a cargo-built debug binary is used, like you would in local development
+      #   - envrionment variables unskip some tests against live systems
       - name: "CLI Integration Tests"
-        env:
-          RUST_BACKTRACE: 1
-        run: nix develop -L --no-update-lock-file --command just integ-tests
-
-      - name: "Functional Tests"
-        if: ${{ github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch' }}
         env:
           RUST_BACKTRACE: 1
           AUTH0_FLOX_DEV_CLIENT_SECRET: "${{ secrets.MANAGED_AUTH0_FLOX_DEV_CLIENT_SECRET }}"
```

Comment:
The tests that use this are failing and I can't see why: https://github.com/flox/flox/actions/runs/9643818820/job/26594589165?pr=1687#step:7:232